soccer/Robot.cpp

```diff
@@ -347,6 +347,7 @@ void OurRobot::_kick(uint8_t strength) {
 }
 
 void OurRobot::_chip(uint8_t strength) {
+    std::cout << "YAYYAYAYAYA" << std::endl;
```

Comment:
plzz

Reply:
we're just really excited about our new chipping capability..